.changeset/twenty-eggs-prove.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-notifications-backend-module-email-notification-processor': major
```

Comment:
```suggestion
'@backstage/plugin-notifications-backend-module-email-notification-processor': patch
```

All new plugins will start at `0.0.0` and this will bump it to `0.0.1` as part of the release process, if we use major you would end up with `1.0.0` which is not what we would want.

Reply:
thanks